### PR TITLE
CPASS-342 Add initial swagger docs for the GET /account/gatekeeperNetwork

### PR DESCRIPTION
**Pull request by flipscholtz**
CPASS-342 Add initial swagger docs for the GET /account/gatekeeperNetwork

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -270,9 +270,64 @@ paths:
         - pass_auth:
             - write:token
             - read:token
+
+  /account/gatekeeperNetwork:
+    get:
+      tags:
+        - gatekeeperNetwork
+      summary: Retrieve a list of all your organization's custom pass types ( gatekeeper networks ).
```
Comment (kevinhcolgan) [inline swagger.yaml:278]: any reason for the spaces in the brackets?

File: swagger.yaml [hunk 2]
```diff
@@ -270,9 +270,64 @@ paths:
...
+      description: Retrieve the full list of all custom pass types ( also called gatekeeper networks ) managed by your organization
```
Comment (kevinhcolgan) [inline swagger.yaml:279]: don't think we need 'all' if we already say 'full list'

File: swagger.yaml [hunk 3]
```diff
@@ -270,9 +270,64 @@ paths:
...
+                      $ref: '#/components/schemas/GatekeeperNetworkResponse'
+      security:
+        - pass_auth:
+            - write:token
+            - read:token
+  /account/gatekeeperNetwork/{id}:
+    get:
+      tags:
+        - gatekeeperNetwork
+      summary: Retrieve a specific pass type ( gatekeeper network ) by id.
```
Comment (kevinhcolgan) [inline swagger.yaml:302]: spaces in brackets

File: swagger.yaml (hunk 2 above)
Comment (flipscholtz) [reply swagger.yaml]: fixed

**Review by kevinhcolgan — APPROVED** [review]

**Pull request merged**